src/tests/radsniff/all.mk

```diff
@@ -15,7 +15,7 @@ $(eval $(call TEST_BOOTSTRAP))
 #
 .PRECIOUS: $(OUTPUT)/%.pcap
 $(OUTPUT)/%.pcap: $(DIR)/%.pcap.gz
-	$(Q)gzcat $< >$@
```

Comment:
any system which has gzip should have gzcat, no?

Reply:
No. 

```
[root@devbox-freeradius-server-ubuntu1910 freeradius-server.git]# gzcat
bash: gzcat: command not found
[root@devbox-freeradius-server-ubuntu1910 freeradius-server.git]#```

That's the reason for `gunzip -c`

Reply:
it's just zcat - but that can probably get confused with non-gzip zcat on older systems.